From ab8376b851ff2cde14fd4ddd6827771f513811d5 Mon Sep 17 00:00:00 2001
From: xtex <xtexchooser@duck.com>
Date: Sun, 15 Sep 2024 21:24:25 +0800
Subject: [PATCH 09/10] FROM ARCHLINUX: fix: python 3.12 compatibility
X-Developer-Signature: v=1; a=openpgp-sha256; l=660; i=xtexchooser@duck.com;
 h=from:subject; bh=HVebMdcsfnXs1rtOhiEsaQPu1dsJh68P9mGDTv5raso=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDGkvVLZ2dk5c8HZvoLxBjNzsF7G2/3w5rrhGyk+8Lu0i3
 d7w1F6yo5SFQYyLQVZMkaXIsMGbVSedX3RZuSzMHFYmkCEMXJwCMJEYHkaGH9GP7hb0cn6zW3zn
 8vTPuuIeD8W/ucXm7F5i56GQpGz7h+GfuWJJRvQl5vS2e52X37ouMbr71o3VxPahg9bhhGo+E2s
 GAA==
X-Developer-Key: i=xtexchooser@duck.com; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

---
 .../Devices/EFI/Firmware/BaseTools/Source/Python/UPT/UPT.py     | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/VBox/Devices/EFI/Firmware/BaseTools/Source/Python/UPT/UPT.py b/src/VBox/Devices/EFI/Firmware/BaseTools/Source/Python/UPT/UPT.py
index 693e01e088e0..35a7c9528ab5 100755
--- a/src/VBox/Devices/EFI/Firmware/BaseTools/Source/Python/UPT/UPT.py
+++ b/src/VBox/Devices/EFI/Firmware/BaseTools/Source/Python/UPT/UPT.py
@@ -15,7 +15,7 @@ UPT
 #
 import locale
 import sys
-from imp import reload
+from impportlib import reload
 encoding = locale.getdefaultlocale()[1]
 if encoding:
     reload(sys)
-- 
2.46.0

